proposal.html

```diff
@@ -37,6 +37,12 @@ <h1>Array.prototype.flatten( [ _depth_ ] )</h1>
     1. Perform ? FlattenIntoArray(_A_, _O_, _sourceLen_, 0, _depthNum_).
     1. Return _A_.
   </emu-alg>
+  <p>Its set accessor function performs the following steps when called with parameter _replacement_:</p>
+  <emu-alg>
+    1. If _replacement_ was not provided, let _replacement_ be *undefined*.
+    1. Perform ? DefinePropertyOrThrow(*this* value, `"flatten"`, PropertyDescriptor { [[Value]]: _replacement_ }).
```

Comment:
`[[Writable]]` etc need to be specified: they otherwise default to [the wrong thing](https://tc39.github.io/ecma262/#table-4).